lib/qa/authorities/linked_data/config.rb

```diff
@@ -44,11 +44,20 @@ def prefixes
         @prefixes ||= authority_config.fetch(:prefixes, {})
       end
 
+      def config_version
+        @config_version ||= authority_config.fetch(:QaConfigVersion, '1.0')
+      end
+
+      def config_version?(version)
+        config_version == version
+      end
```

Comment:
Adds support for versioning to the linked data config files.  NOTE: Defaults to 1.0 if not specified for backward compatibility.